src/Native/pkg/runtime.native.System.Data.SqlClient.sni/runtime.native.System.Data.SqlClient.sni.pkgproj

```diff
@@ -4,13 +4,14 @@
   <PropertyGroup>
     <SkipPackageFileCheck>true</SkipPackageFileCheck>
     <SkipValidatePackage>true</SkipValidatePackage>
+    <PreReleaseLabel>servicing</PreReleaseLabel>
   </PropertyGroup>
   <ItemGroup>
     <Dependency Include="runtime.win7-x64.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion)</Version>
+      <Version>4.4.0-beta-24612-00</Version>
     </Dependency>
     <Dependency Include="runtime.win7-x86.runtime.native.System.Data.SqlClient.sni">
-      <Version>$(RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion)</Version>
+      <Version>4.4.0-beta-24612-00</Version>
```

Comment:
Yes lets try to update the RuntimeWin7RuntimeNativeSystemDataSqlClientSniVersion property instead.